nncf/torch/__init__.py

```diff
@@ -76,4 +77,5 @@
 if torch.__version__ >= "2.5.0":
     from torch._dynamo.polyfills import loader
 
-patch_torch_operators()
+if os.getenv("NNCF_EXPERIMENTAL_TORCH_TRACING") is None:
+    patch_torch_operators()
```

Comment:
No, now it's will be used only for experimental features.
After switching algorithms to new tracing as default way, it will be removed or renamed. I'm not sure that it will still exists in next releases. 
It's more temporal solution to have possible reuse some code from nncf/torch, than introducing new features of nncf